src/antithesis_sdk/_internal/handlers.py

```diff
@@ -151,3 +152,18 @@ def handles_output(self) -> bool:
 
 def _setup_handler() -> Handler:
     return VoidstarHandler.get() or LocalHandler.get() or NoopHandler.get()
+
+def _version_message() -> str:
```

Comment:
I do vaguely believe that this belongs where it was previously in internal as creating a version message isn't inherent;y tied to handlers

Reply:
`_version_message` was moved into handlers to avoid a cyclic import.  There could be a better way, and it can be updated if/when required.  If it isn't crucial to move `_version_message` can it stay here for now?

Reply:
Ah, I believe you can avoid cyclic import by shuffling around the order in which it is loaded (not first thing upon module load), but not too important, lets keep it here!